ToggleButton/ToggleButton.js

```diff
@@ -150,18 +148,19 @@ const ToggleButtonBase = kind({
 			}
 			return c;
 		},
-		iconToggleButton: ({children, toggleOnLabel, toggleOffLabel, icon}) => (!children && !toggleOnLabel && !toggleOffLabel && icon)
+		iconToggleButton: ({children, toggleOnLabel, toggleOffLabel, icon}) =>
+			!children && !toggleOnLabel && !toggleOffLabel && icon
 	},
 
 	render: ({children, iconToggleButton, icon, selected, ...rest}) => {
 		delete rest.underline;
 		delete rest.toggleOffLabel;
 		delete rest.toggleOnLabel;
 
-		return iconToggleButton ? (
-			<IconButton {...rest} aria-pressed={selected} css={css} selected={selected}>{icon}</IconButton>
-		) : (
-			<Button {...rest} aria-pressed={selected} css={css} icon={icon} selected={selected}>{children}</Button>
+		return (
+			<Button {...rest} aria-pressed={selected} css={css} icon={icon} selected={selected}>
+				{children}
```

Comment:
I think we need to incorporate the `if (iconToggleButton)` logic in the props of this button, maybe?

Something like
```
<Button {...rest} aria-pressed={selected} css={css} icon={iconToggleButton ? icon : null} selected={selected}>
    {iconToggleButton ? null : children}
</Button>
```
Or, if we know that we want to potentially have both, remove the computed `iconToggleButton` completely.

Reply:
Removed iconToggleButton logic as button already has the logic to handle the icon with children.